components/popover/popover.vue

```diff
@@ -267,6 +267,14 @@ export default {
       validator: contentWidth => POPOVER_CONTENT_WIDTHS.includes(contentWidth),
     },
 
+    /**
+     * Whether to apply transition on initial render in the content lazy show component.
+     */
+    contentAppear: {
+      type: Boolean,
+      default: false,
+    },
```

Comment:
hmm wondering if we can know this automatically if the component has the open prop set to true when created

Reply:
tried that but it doesn't execute the transition hooks anyways